packages/AlgernonTests.package/ALGTestFuzzySearch.class/instance/testScore0CaseInsensitive.st

```diff
@@ -0,0 +1,5 @@
+testing
+testScore0CaseInsensitive
+	"Make sure normal input works"
+	self runQuery: 'aLGERNON'.
```

Comment:
Should we test for the position of the item as well ? (Regards all testScore-Tests)

Reply:
Theoretically the testScore test cover the ranking, as it is not explicit but derived from that. Also that would be more a test of the SmartSorter as well.

What do you think?